pkg/providers/instancetype/types.go

```diff
@@ -203,6 +203,7 @@ func memory(ctx context.Context, info *ec2.InstanceTypeInfo) *resource.Quantity
 }
 
 // Setting ephemeral-storage to be either the default value, what is defined in blockDeviceMappings, or the combined size of local store volumes.
+// nolint:gocyclo
```

Comment:
Since you didn't change anything in this function, do you still need this `nolint` statement?